packages/salesforcedx-apex-debugger/test/unit/commands/baseDebuggerCommand.test.ts

```diff
@@ -32,7 +32,7 @@ export function getDefaultHeaders(contentLength: number): any {
     'Content-Type': 'application/json;charset=utf-8',
     Accept: 'application/json',
     Authorization: `OAuth 123`,
-    'Content-Length': contentLength,
+    'Content-Length': String(contentLength),
```

Comment:
request-light bump altered the type here to be a string so this fixes all the tests that use this header.